src/playbooks/site.yml

```diff
@@ -135,6 +135,7 @@
     - base-extras
     - imagemagick
     - apache-php
+    - netdata
```

Comment:
Change this from just

```yaml
- netdata
```

to

```yaml
- role: netdata
  when: m_install_netdata
```

Then, above copy the `firewall_port` role for `load_balancers_all` for port 19999, since that will need to be opened on the netdata servers for requests from load balancers. This I think:

```yaml
    - role: firewall_port
      firewall_action: open
      firewall_port: 19999
      firewall_protocol: tcp
      firewall_servers: "{{ load_balancers_all }}"
      firewall_zone: "{{m_private_networking_zone|default('public')}}"
```